examples/examplesVersionDependency.ttl

```diff
@@ -11,6 +11,16 @@
 @prefix ver: <https://w3id.org/semanticarts/ns/ontology/versioning/> .
 @prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
 
+<https://ontologies.exco.com/exeoAilments>
```

Comment:
Would make more sense if all the ontologies were named according to the error or test case they relate to, as in the last two.

Reply:
> Would make more sense if all the ontologies were named according to the error or test case they relate to, as in the last two.

@rjyounes Good question, after some thought, I prefer to leave it as it is because they demonstrate a realistic use of imports and dependency. The others are there _only_ for the purpose of demonstrating a particular error.